redisinsight/ui/src/components/global-url-handler/GlobalUrlHandler.tsx

```diff
@@ -29,8 +32,18 @@ const GlobalUrlHandler = () => {
   const history = useHistory()
   const dispatch = useDispatch()
   const location = useLocation()
+  const searchParams = new URLSearchParams(location.search)
```

Comment:
move please `const returnUrl =...` and `dispatch(setReturnUrl(...))` to the useEffect on the 81th line